### PR TITLE
Add request id header

### PR DESCRIPTION
**Pull request by ewalker11**

### CODE REVIEW

File: library/Zend/Http/Client.php [hunk 1]
```diff
@@ -1192,6 +1192,13 @@ protected function _prepareHeaders()
             }
         }
 
+        // Set the request id header if the unique id environment variable
+        // is set.
+        $uniqueId = getenv('UNIQUE_ID');
+        if ($uniqueId && !isset($this->headers['x-request-id'])) {
+            $headers[] = "x-request-id: {$uniqueId}";
```
Comment (blockjon) [inline library/Zend/Http/Client.php:1199]: On the previous line you are referring to $this->headers and then on the following line just $headers

Gotta sort that out

Comment (blockjon) [inline library/Zend/Http/Client.php:1199]: Or... maybe I'm missing something?

Comment (blockjon) [inline library/Zend/Http/Client.php:1199]: +1 if it works

Comment (ewalker11) [inline library/Zend/Http/Client.php:1199]: So this method forms a $headers array in which it places a couple of headers (like this one). Eventually, it rolls the full $this->headers array into the $headers array and then returns it.

**Pull request merged**